processing/src/main/java/org/apache/druid/query/expression/LookupExprMacro.java

```diff
@@ -116,4 +126,15 @@ public void decorateCacheKeyBuilder(CacheKeyBuilder builder)
 
     return new LookupExpr(arg);
   }
+
+  private String getReplaceMissingValueWith(final List<Expr> args)
+  {
+    if (args.size() > 2) {
+      final Expr missingValExpr = args.get(2);
+      if (missingValExpr.isLiteral() && missingValExpr.getLiteralValue() != null) {
+        return missingValExpr.getLiteralValue().toString();
+      }
+    }
+    return null;
```

Comment:
I think if you are not going to handle a dynamic argument for the 'missing value' parameter, this should be an exception if the argument is not a literal. There is `validationHelperCheckArgIsLiteral` that can be called if we only want to support this argument being a constant.

Silently returning null seems incorrect

Reply:
added validationHelperCheckArgIsLiteral check